install/charts/templates/karydia-default-network-policy.sh.tpl

```diff
@@ -31,6 +31,14 @@ spec:
   podSelector: {}
   policyTypes:
   - Egress
+  egress:
+  - to:
+    - ipBlock:
+        cidr: 0.0.0.0/0
+        except:
+        - 10.250.0.0/16      # host network IP for AWS
```

Comment:
Can we be sure of only blocking the scoped use case, e.g. `host network IP for AWS`, if we define such a complete IP range? So that nothing else breaks?

Reply:
?

Reply:
@CodeClinch @SAPJonasZengerle 
When Christian and I met the Gardener colleagues, they told us that it is the right solution and it is the right source for the information.